app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function should verify that the command string contains exactly three elements and starts with 'cp'. This is necessary to ensure the command is valid according to the task description and checklist.